derive/src/lib.rs

```diff
@@ -197,6 +198,7 @@ pub fn decode_derive(input: proc_macro::TokenStream) -> proc_macro::TokenStream
 
 	let input_ = quote!(__codec_input_edqy);
 	let decoding = decode::quote(&input.data, name, &input_);
+	let skipping = skip::quote(&input.data, name, &input_);
```

Comment:
```suggestion
	let skip_impl = skip::quote(&input.data, name, &input_);
```

maybe? (Same for `decoding`)